attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto

```diff
@@ -99,6 +99,7 @@ message AttributeMetadata {
   string id = 14;
   bool groupable = 15;
   AttributeDefinition definition = 16;
+  string scope_string = 17;
```

Comment:
Starting to replace the enum scope with a string version to remove requirement of compile-time knowledge of scopes.